### PR TITLE
Add sum integration

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Add sum integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/79398
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3717
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sum.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ha_integration_type: helper
+---
+
+The Sum integration consumes the state from other sensors to calculate the sum of it's states.
+
+If any source sensor provides an unknown state, this sensor will also be in unknown state.
+
+If the unit of measurement of the sensors differs, the Sum sensor will continue to calculate the sum but provide a warning in the log.
+
+{% include integrations/config_flow.md %}
```
Comment (frenck) [inline source/_integrations/sum.markdown:26]: This is a helper, I would expect YAML configuration to be documented?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/sum.markdown (hunk 1 above)
Comment (gjohansson-ST) [reply source/_integrations/sum.markdown:26]: I did not include yaml support but given your comment I suppose then for helpers yaml support is required always?

Comment (frenck) [reply source/_integrations/sum.markdown:26]: I think it makes sense, as we do that for all helpers at this point (it also doesn't connect to a device or service).

Comment (gjohansson-ST) [reply source/_integrations/sum.markdown:26]: Yes. I'll add it in

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request closed without merging**